### ISSUE TITLE
simplemde update in ChatterPostController

### ISSUE THREAD

**Issue by dodgydre** [thedevdojo/chatter#144]
It seems that the `$post->body = Purifier::clean($request->body);` on the update method is adding html tags to the post body when updated in the database if using markdown editor. 

This then causes rendering issues on post display.

Updating the update() method to the following seems to fix it for me.
```
if ($post->markdown) {
  $post->body = $request->body;
} else {
  $post->body = Purifier::clean($request->body);
}
```

Is this a correct tweak or should the Purifier not be doing this to me?

Thanks

**Comment by galdazbiz** [comment]
I don't think the devs are very into this project, but thanks for helping

**Comment by marktopper** [comment]
We are open to pull requests.

**Comment by marktopper** [comment]
Fixed in #193
